src/chronometer.js

```diff
@@ -1,33 +1,52 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
     // ... your code goes here
-  }
+      this.intervalId = setInterval(() => {
+          this.currentTime++;
+          if (printTimeCallback) {
+            printTimeCallback();
+          }
+      }, 1000);
+    }
 
   getMinutes() {
     // ... your code goes here
+    let minutes = Math.floor(this.currentTime/60);
+    return minutes;
   }
 
   getSeconds() {
     // ... your code goes here
+    let seconds = Math.floor(this.currentTime%60);
+    return seconds;
   }
 
   computeTwoDigitNumber(value) {
     // ... your code goes here
+    if (value.toString.length===1){
+      return value.toString().padStart(2,"0");
```

Comment:
If you want to save some lines of code, you could use `string interpolation`.
Something like this:
```
computeTwoDigitNumber(value) {
    if (value < 10) return `0${value}`;
    else return `${value}`;
  }
```